modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -64,38 +52,20 @@ Before recovering the VGs from the previous {lvms} installation, ensure that the
 |`object`
 |Contains the configuration to choose the nodes on which you want to create the LVM volume group. If this field is empty, all nodes without no-schedule taints are considered.
 
-[NOTE]
-====
 On the control-plane node, {lvms} detects and uses the additional worker nodes when the new nodes become active in the cluster. 
-====
 
 |`nodeSelector.nodeSelectorTerms`
 |`array`
 |Configure the requirements that are used to select the node.
 
 |`deviceClasses.deviceSelector`
 |`object`
-|Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group, and force wipe the devices that are added to the LVM volume group. 
-
-You can specify the device paths in the `paths` field, the `optionalPaths` field, or both. If you do not specify the device paths in both `paths` and `optionalPaths`, {lvms} adds the supported unused devices to the LVM volume group.
-
-[IMPORTANT]
-====
-After a device is added to the LVM volume group, it cannot be removed.
-==== 
-
-{lvms} adds the devices to the LVM volume group only if the following conditions are met:
-
-* The device path exists.
-
-* The device is supported by {lvms}.
+|Contains the configuration to perform the following actions:
 
-[NOTE]
-====
-The devices that you want to add to the volume group must be supported by {lvms}. For information about unsupported devices, see _Devices not supported by {lvms}_ in _Additional resources_.
-====
+* Specify the paths to the devices that you want to add to the LVM volume group.
+* Force wipe the devices that are added to the LVM volume group.
 
-You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. For more information, see _Integrating RAID arrays with {lvms}_ in _Additional resources_. 
+For more information, see "About adding devices to a volume group" in the "Additional resources" section.
```

Comment:
Here the cited reference is done with " " and switched from _ _ formatting.

Reply:
Updated as per the suggestion.